lib/ansible/modules/apt.py

```diff
@@ -242,7 +242,7 @@
     update_cache: yes
 
 - name: Install the version '1.18.0' of package "nginx" and allow potential downgrades
-  apt:
+  ansible.builtin.apt:
```

Comment:
looks like all the rest of the apt examples in this file still use the short name. Can you update to FQCN as well please? Alternately I can merge https://github.com/ansible/ansible/pull/76802/files and you can rebase?

Reply:
Re-added in my changes, I'm not sure why they were wiped during the rebase. 
Thanks for the second glance over